src/components/DocPage/DocPage.scss

```diff
@@ -338,9 +338,3 @@
     }
     /* stylelint-enable declaration-no-important */
 }
-
-.mobile {
```

Comment:
Please add a few tests for this behavior, we've been catching problems here too often lately.